code/script.js

```diff
@@ -1,9 +1,14 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
-}
+var acc = document.getElementsByClassName("accordion");
```

Comment:
I don't think you need this file, hmm? ;) You use `app.js` in HTML

Reply:
True. Removed. ;)